source/ConditionsStore.cpp

```diff
@@ -204,28 +204,30 @@ void ConditionsStore::Save(DataWriter &out) const
 // derived from other data-structures (derived conditions).
 int64_t ConditionsStore::Get(const string &name) const
 {
-	const ConditionEntry *ce = GetEntry(name);
+	const string expanded = Expand(name);
```

Comment:
This Expand now gets done for _every_ condition access. There are however some plans to use conditions also in the realtime / at-flight-time parts of the game.

Reply:
`Expand` is really fast when there is no expansion to perform (it's basically just a single iteration over the string, which we do anyway for lookups).

I could add another getter function that doesn't expand entries if it becomes necessary, though I doubt it's a big difference in performance.